docs/modules/ROOT/pages/archetype.adoc

```diff
@@ -81,6 +81,37 @@ General pattern: ``<internet domain name in reverse>.<your area name>``
 
 | Yes
 
```

Comment:
i: rest of the fields were in alphabetical order so moved `licenseName` up to match